src/chef-server-ctl/bin/chef-server-ctl

```diff
@@ -120,9 +120,12 @@ module Omnibus
     end
 
     def postgresql_connection(postgres)
-      PGconn.open('user' => postgres['db_superuser'], 'host' => postgres['vip'],
-                  'password' => credentials.get('postgresql', 'db_superuser_password'),
-                  'port' => postgres['port'], 'dbname' => 'template1')
+      PG::Connection.open('user' => postgres['db_superuser'],
+                          'host' =>       postgres['vip'],
+                          'password' =>   credentials.get('postgresql', 'db_superuser_password'),
+                          'port' =>       postgres['port'],
+                          'sslmode' =>    postgres['sslmode'],
```

Comment:
Based on https://www.rubydoc.info/gems/pg/PG/Connection the sslmode might have to be options. 
`'options' => postgres['sslmode']`